awscli/help.py

```diff
@@ -259,6 +272,39 @@ def event_class(self):
     def name(self):
         return self.obj.name
 
+    @property
+    def topic_table(self):
+        if self._topic_table is None:
+            if self._topic_tag_db is None:
+                self._topic_tag_db = TopicTagDB()
+            self._topic_tag_db.load_json_index()
+            self._topic_table = self._create_topic_table()
+        return self._topic_table
+
+    def _create_topic_table(self):
+        topic_table = {}
+        # Add the ``aws help topics`` command to the ``topic_table``
+        topic_lister_command = TopicListerCommand(
+            self.session, self._topic_tag_db)
+        topic_table['topics'] = topic_lister_command
+        topic_names = self._topic_tag_db.get_all_topic_names()
+
+        # Add all of the possible topics to the ``topic_table``
+        for topic_name in topic_names:
+            topic_help_command = TopicHelpCommand(
+                self.session, topic_name, self._topic_tag_db)
+            topic_table[topic_name] = topic_help_command
+        return topic_table
+
+    def __call__(self, args, parsed_globals):
+        if args:
+            topic_parser = ArgTableArgParser({}, self.topic_table)
+            parsed_topic, remaining = topic_parser.parse_known_args(args)
+            self.topic_table[parsed_topic.subcommand].__call__(
```

Comment:
You don't need the `__call__` here.  Function invocation implicitly calls ``__call__``.